reader_core/src/gen7/reader.rs

```diff
@@ -3,19 +3,29 @@ use crate::pnp;
 use core::num::{NonZeroU8, NonZeroU32};
 use pkm_rs::{Pk7, PokeCrypto};
 
+pub const SM_SOS_BASE_ADDR: u32 = 0x30038c34;
+pub const USUM_SOS_BASE_ADDR: u32 = 0x30038e20;
+pub const SM_SOS_SFMT_ADDR: u32 = SM_SOS_BASE_ADDR + 0x10;
+pub const USUM_SOS_SFMT_ADDR: u32 = USUM_SOS_BASE_ADDR + 0x10;
+pub const PKM_CONT_SIZE: u32 = 0x330;
+pub const PK7_DATA_SIZE: u32 = 0x1E4;
```

Comment:
PKHeX's source says the sizes of pk7s are 0x104 or 0xe8, so this looks more like space between two Pokemon, not the size themselves. Let's rename this to reflect that.

It might be the size of the the pk7 with its container – if we can prove that it's an alright label too.